### ISSUE TITLE
Resource 22: Resource Temporarily unavailable - connecting to RasPi Servers 

### ISSUE THREAD

**Issue by garfbradaz** [Microsoft/WSL#3136]
## Windows build number:  
Microsoft Windows [Version 10.0.16299.371]

## What Am I doing and what's happening:  

I have created a 6 Pi Cluster (They run Stretch). I've managed to SSH using WSL Ubuntu before. I have now rebooted the machines and now I cannot SSH back in and I get:

> ssh: connect to host workerpi5 port 22: Resource temporarily unavailable

I cannot SSH onto any of the Pi's now. I'm running:

> ssh   -v <name>workerpi5

Here is the debug output.

> OpenSSH_7.2p2 Ubuntu-4ubuntu2.4, OpenSSL 1.0.2g  1 Mar 2016
> debug1: Reading configuration data /etc/ssh/ssh_config
> debug1: /etc/ssh/ssh_config line 19: Applying options for *
> debug1: Connecting to workerpi5 [192.168.8.200] port 22.
> debug1: connect to address 192.168.8.200 port 22: Resource temporarily unavailable
> ssh: connect to host workerpi5 port 22: Resource temporarily unavailable

## What's wrong / what should be happening instead:

I should be able to SSH into my Pi's using WSL.

## What have I tried?

- Rebooted the Pi's again.
- Run package updates/upgrades for WSL.
- Re-installed openssh-client on WSL.
- I can ping the machines fine from WSL:

![image](https://user-images.githubusercontent.com/959766/39449201-552d5f6e-4cbf-11e8-9c3d-fa28c517dc7a.png)

- Port 22 is open within Windows Firewall.

## Strace of the failing command

[mytrace](https://gist.github.com/garfbradaz/32455d3fd9b157bb9d8240e992204ca4)

**Comment by DarthSpock** [comment]
See #714.

**Comment by garfbradaz** [comment]
> See #714.
@DarthSpock 👍  - Do I need to just copy my issue into #714 ?

**Comment by DarthSpock** [comment]
I would just comment on there and ref this issue as you already have a trace here.

**Comment by Biswa96** [comment]
GitHub already connects the reference issue.

**Comment by garfbradaz** [comment]
Hey Guys,

I did a lot of digging today and found the Router I have incorrectly was showing my static IPs were still leased to my Pi's, which when I debugged the syslog on them, they were not and the SSH packets were failing Router level. I've refreshed my router (not a reboot as that didnt work), but actually stuck the data ethernet from router into each Pi, and now working (Ive got the Pi's hooked into a Switch).

Apologies for this!
